homeassistant/components/flux_led/light.py

```diff
@@ -92,39 +128,36 @@
     EFFECT_WHITE_STROBE: 0x37,
     EFFECT_COLORJUMP: 0x38,
 }
-EFFECT_CUSTOM_CODE = 0x60
-
-TRANSITION_GRADUAL = "gradual"
-TRANSITION_JUMP = "jump"
-TRANSITION_STROBE = "strobe"
+EFFECT_ID_NAME: Final = {v: k for k, v in EFFECT_MAP.items()}
+EFFECT_CUSTOM_CODE: Final = 0x60
+
+WHITE_MODES: Final = {MODE_RGBW, MODE_RGBWW, MODE_RGBCW}
```

Comment:
@icemanch Please check this logic to make sure it makes sense

Reply:
Trying to use RGBWW or RGBCW is going to add a lot of unnecessary code that's going to get overwritten.   I removed both as there really isn't a good way for Home Assistant to figure out if it is RGBWW capable.